### PR TITLE
feat: added hamburger in all pages

### PR DESCRIPTION
**Pull request by tanishq20**
Added hamburger for components showcase in mobile view

Live preview link: https://deploy-preview-37--compact-ui.netlify.app/

[Issue no. 8](https://github.com/tanishq20/compact-UI/issues/36)

preview image

### CODE REVIEW

**Review by shashidwivedi — APPROVED** [review]
Looks clean, can be merged

File: docs/stylesheets/main.css [hunk 1]
```diff
@@ -114,6 +114,45 @@ body {
...
+  cursor: pointer;
+  position: absolute;
+  top: 0;
+  right: 25px;
+  font-size: 3.6rem;
+}
+
+.sm-sidebar-header {
+  background-color: var(--secondary-color);
+  color: var(white);
```
Comment (aishwarya-bm) [inline docs/stylesheets/main.css]: Here your variable name should be --white?

**Review by aishwarya-bm — APPROVED** [review]
Clean code, good to merge

File: docs/stylesheets/main.css (hunk 1 above)
Comment (tanishq20) [reply docs/stylesheets/main.css]: Thanks for pointing out that Aishwarya.

Comment (tanishq20) [reply docs/stylesheets/main.css]: Fixed

**Pull request merged**